recipe/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "python-ffmpeg" %}
+{% set version = "2.0.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/jonghwanhyeon/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz"
```

```suggestion
  url: https://github.com/jonghwanhyeon/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
```
Redundant double quotes `"`